.gitignore

```diff
@@ -25,3 +25,7 @@ data/
 pkg/web/static/tailwind.css
 tmp/
 mentions.json
+node_modules/
+package.json
+package-lock.json
```

Comment:
```suggestion
```
Now that the tailwind CLI is being used, can this be removed from the gitignore?